core/tests/protractor_utils/forms.js

```diff
@@ -294,24 +301,29 @@ var SetOfTranslatableHtmlContentIdsEditor = function(elem) {
 var UnicodeEditor = function(elem) {
   return {
     setValue: async function(text) {
-      await elem.element(by.tagName('input')).clear();
-      await elem.element(by.tagName('input')).sendKeys(text);
+      var unicodeEditorInput = elem.element(by.tagName('input'));
+      await action.clear('Unicode Editor Input', unicodeEditorInput);
+      await action.sendKeys('Unicode Editor Input', unicodeEditorInput, text);
     }
   };
 };
 
 var AutocompleteDropdownEditor = function(elem) {
   return {
     setValue: async function(text) {
-      await elem.element(by.css('.select2-container')).click();
+      var select2Container = elem.element(by.css('.select2-container'));
+      await action.click('Select2 Container', select2Container);
       // NOTE: the input field is top-level in the DOM, and is outside the
       // context of 'elem'. The 'select2-dropdown' id is assigned to the input
       // field when it is 'activated', i.e. when the dropdown is clicked.
-      await element(by.css('.select2-dropdown')).element(
-        by.css('.select2-search input')).sendKeys(text + '\n');
+      var select2SearchInput = element(
+        by.css('.select2-dropdown')).element(by.css('.select2-search input'));
+      await action.sendKeys(
+        'Select2 Search Input in Dropdown', select2SearchInput, text + '\n');
     },
     expectOptionsToBe: async function(expectedOptions) {
-      await elem.element(by.css('.select2-container')).click();
+      var select2Container = elem.element(by.css('.select2-container'));
+      await action.click('Select2 Container', select2Container);
       var actualOptions = await element(by.css('.select2-dropdown'))
         .all(by.tagName('li')).map(
           async function(optionElem) {
```

Comment:
Below, please add a waitFor before the .getText. You can also you our new `action.getText` function!

Reply:
done!